akka-actor/src/main/scala/akka/util/ZipfianGenerator.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util
+
+/**
+ * Zipfian generator algorithm from:
+ * "Quickly Generating Billion-Record Synthetic Databases", Jim Gray et al.
+ */
+private[akka] object ZipfianGenerator {
```

Comment:
should this be placed in test sources?

Reply:
yes, moved in https://github.com/akka/akka/pull/31078/commits/95221ccd42bb68b10fd9b6260f7e6edc46668b8c